spec/revisement/changeset/file_modification_spec.rb

```diff
@@ -124,7 +126,7 @@
                 - pp
                 current_revision:
                 - m
-                - n
+                - "n"
```

Comment:
Is this a behavior change?

Reply:
One I couldn't figure out... Stepping through with the debugger in `file_modification.rb`:

```ruby
prior
{"a"=>"bbcc", "d"=>["e", "f", "g"], "h"=>{"i"=>{"j"=>"k", "l"=>["mm", "nn", "oo", "pp"]}}}

current
{"a"=>"bc", "d"=>["e", "f", "g"], "h"=>{"i"=>{"j"=>"k", "l"=>["m", "n", "o", "p"]}}}

diff.to_yaml...
"/file.m changed at keypath []\n---\na:\n  prior_revision: bbcc\n  current_revision: bc\nh:\n  i:\n  ...\n      - oo\n      - pp\n      current_revision:\n      - m\n      - \"n\"\n      - o\n      - p\n"
```
  
Notice the quotes that got thrown around the `\n`. When I change the original keys to be something else that doesn't include `n`, it still happens: 

```ruby
prior
{"a"=>"bbcc", "d"=>["e", "f", "g"], "h"=>{"i"=>{"j"=>"k", "l"=>["ww", "xx", "yy", "zz"]}}}

current
{"a"=>"bc", "d"=>["e", "f", "g"], "h"=>{"i"=>{"j"=>"k", "l"=>["w", "x", "y", "z"]}}}

diff.to_yaml...
"/file.m changed at keypath []\n---\na:\n  prior_revision: bbcc\n  current_revision: bc\nh:\n  i:\n    l:\n      prior_revision:\n      - ww\n      - xx\n      - yy\n      - zz\n      current_revision:\n      - w\n      - x\n      - \"y\"\n      - z\n"
```

I had thought maybe this was a newline thing, but the second example doesn't support it. It's not event consistently the 2nd element of the array (tested this out appending and prepending elements):

```ruby
      Diff:
       @@ -16,6 +16,6 @@
              - a
              - w
              - x
       -      - y
       +      - "y"
              - z
```

Since wrapping strings in quotations is _usually_ harmless, I moved on from this to continue with the rest of the upgrade.